app/src/main/java/com/example/test/automationtest/util/HogeUtil.java

```diff
@@ -0,0 +1,22 @@
+package com.example.test.automationtest.util;
+
+public class HogeUtil {
+
+    public String hoge(int arg){
+        String reutrnValeu="";
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/4hin/android-automation-test/blob/de7a7d2b3f481daaabc82a215c77c2d4ffd0bc7d/app/src/main/java/com/example/test/automationtest/util/HogeUtil.java#L6"></span>WhitespaceAround: ‘=’ is not followed by whitespace. Empty blocks may only be represented as {} when not part of a multi-block statement (4.1.3)</td>
    </tr>
  </tbody>
</table>